src/wombats/sockets/game.clj

```diff
@@ -139,35 +130,31 @@
     (doseq [channel-id channel-ids]
       (send-fn channel-id))))
 
-(defn- get-message
-  [msg-type payload]
-  {:meta {:msg-type msg-type}
-   :payload payload})
-
 ;; Broadcast/send functions
 
 (defn- send-arena
   [arena]
-  (fn [chan-id]
-    (send-message chan-id
-                  (get-message :frame-update
-                               arena))))
+  (let [msg (m/arena-message arena)]
```

Comment:
this outer let statement may not be necessary. You could just call `arena-message` in place of `msg` since you're closing over arena already